src/styles/_mixins.scss

```diff
@@ -0,0 +1,14 @@
+@mixin display-flex-row() {
```

Comment:
Thank you for creating a mixins file and as well implementing mixins into our regular css file.